client/state/posts/likes/test/reducer.js

```diff
@@ -0,0 +1,307 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import {
+	POST_LIKES_RECEIVE,
+	POST_LIKES_REQUEST,
+	POST_LIKES_REQUEST_FAILURE,
+	POST_LIKES_REQUEST_SUCCESS,
+	SERIALIZE,
+	DESERIALIZE
+} from 'state/action-types';
+import reducer, { requesting, items } from '../reducer';
+import { useSandbox } from 'test/helpers/use-sinon';
+
+describe( 'reducer', () => {
+	useSandbox( ( sandbox ) => {
+		sandbox.stub( console, 'warn' );
+	} );
+
+	it( 'should include expected keys in return value', () => {
+		expect( reducer( undefined, {} ) ).to.have.keys( [
+			'requesting',
+			'items'
+		] );
+	} );
+
+	describe( 'requesting()', () => {
+		it( 'should default to an empty object', () => {
+			const state = requesting( undefined, {} );
+
+			expect( state ).to.eql( {} );
+		} );
+
+		it( 'should set site ID, post ID to true value if a request is initiated', () => {
+			const state = requesting( undefined, {
+				type: POST_LIKES_REQUEST,
+				siteId: 12345678,
+				postId: 50,
+			} );
+
+			expect( state ).to.eql( {
+				12345678: {
+					50: true
+				}
+			} );
+		} );
+
+		it( 'should accumulate the requested site IDs', () => {
+			const state = requesting( deepFreeze( {
+				12345678: {
+					50: true
+				}
+			} ), {
+				type: POST_LIKES_REQUEST,
+				siteId: 87654321,
+				postId: 10,
+			} );
+
+			expect( state ).to.eql( {
+				12345678: {
+					50: true
+				},
+				87654321: {
+					10: true,
+				}
+			} );
+		} );
+
+		it( 'should accumulate the requested post IDs', () => {
+			const state = requesting( deepFreeze( {
+				12345678: {
+					50: true
+				}
+			} ), {
+				type: POST_LIKES_REQUEST,
+				siteId: 12345678,
+				postId: 10,
+			} );
+
+			expect( state ).to.eql( {
+				12345678: {
+					50: true,
+					10: true,
+				}
+			} );
+		} );
+
+		it( 'should set requesting to false if request finishes successfully', () => {
+			const state = requesting( deepFreeze( {
+				12345678: {
+					50: true,
+				}
+			} ), {
+				type: POST_LIKES_REQUEST_SUCCESS,
+				siteId: 12345678,
+				postId: 50,
+			} );
+
+			expect( state ).to.eql( {
+				12345678: {
+					50: false
+				}
+			} );
+		} );
+
+		it( 'should set site ID to false if request finishes unsuccessfully', () => {
```

Comment:
Should this test description read `should set post ID to false if request finishes unsuccessfully`?